### PR TITLE
Rollbacks pg14 test release changes

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: templates/Dockerfile-rpm.tmpl [hunk 1]
```diff
@@ -35,12 +35,10 @@ RUN if [[ %%os%%   == oraclelinux ]] && [[ %%release%% == 7 ]]; then yum install
         && rpm -ivh lz4-devel-r131-1.el6.x86_64.rpm;  \
         else yum install -y lz4 lz4-devel; fi
 
-# Enable testing repo if postgres version is 14
```
Comment (gurkanindibay) [inline templates/Dockerfile-rpm.tmpl:38]: Removed test repo changes and unified the command with the next image

File: templates/Dockerfile-deb.tmpl [hunk 2]
```diff
@@ -38,11 +38,8 @@ RUN set -ex; \
     apt-key list
 
 # install build tools and PostgreSQL development files
-# Since pg 14 is not released yet, pg 14 repo is needed to add  to install pg 14 related libraries.
-# Since postgresql-server-dev-all library does not include pg14 libraries, postgresql-server-dev-14
-# library is needed to add.
-# TODO After Pg14 official release the changes should be rolled back.
-RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main 14' > /etc/apt/sources.list.d/pgdg.list \
+
+RUN echo 'deb http://apt.postgresql.org/pub/repos/apt/ %%release%%-pgdg main' > /etc/apt/sources.list.d/pgdg.list \
```
Comment (gurkanindibay) [inline templates/Dockerfile-deb.tmpl:42]: Remove test repo

**Review by onurctirtir — APPROVED** [review]
Compared with the changes made in https://github.com/citusdata/packaging/pull/753/ & https://github.com/citusdata/packaging/pull/747 and all LGTM.

**Comment by hanefi** [comment]
Please rebase

**Review by hanefi — APPROVED** [review]

**Pull request merged**